### PR TITLE
beach ruin, The Treasure Cove!

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
This PR adds a new Beach ruin, Treasure Cove. 

image

Loot: 
cm-16 rifle (main loot)
energy gun
pirate sabre
frontiersmen hardsuit
misc combat supplies
secret documents
2x abandoned crates
research note / tesla researcher
basic engineering supplies (smes/tools/autolathe/battery charger)
two boats
silver crate / hidden gold crate
misc junk
______
Threat: 
1x spacesuit ranged pirate
2x sword pirates
1x ranged pirate
punji sticks
_____

Lore tidbit:
This "humble abode" is the home of our 5- now 4 Pirate friends! After a mildly successful raid on a CMM VIP transport, they managed to take a Cargo tech (the VIP), and a CMM guard as hostage. sadly it didn't all go as planned, and the CMM officer managed to free himself and killed one of the pirates. This is where you now find the cave, with both hostages executed, their brother buried, and the pirates grieving his unfortunate passing. 

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
more ruins = good.

## Changelog

:cl:
add: Adds a new beach ruin, the beach_treasure_cove
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/BeachRuins/beach_treasure_cove.dmm [hunk 1]
```diff
@@ -0,0 +1,3215 @@
...
+/obj/effect/turf_decal/siding/wood{
+	dir = 10
+	},
+/obj/structure/curtain/bounty,
+/obj/structure/fluff/hedge,
+/turf/open/floor/wood,
+/area/ruin/beach/treasure_cove)
+"pz" = (
+/turf/closed/wall/concrete{
+	desc = "As you draw near the ancient wall, a sense of foreboding overcomes you. You aren't sure why, but you feel this dusty structure may contain great dangers."
```
Comment (Erikafox) [inline _maps/RandomRuins/BeachRuins/beach_treasure_cove.dmm]: this desc is off the complex iirc. Intentional?

File: code/game/area/areas/ruins/beachplanet.dm [hunk 2]
```diff
@@ -12,3 +12,11 @@
 /area/ruin/beach/complex/shuttle
 	name = "Syndicate Shuttle"
 	icon_state = "green"
+
+
+
+//beach_treasure_cove, the beeginning
+
+/area/ruin/beach/treasure_cove
+	name = "Quirky Cavern"
```
Comment (Erikafox) [inline code/game/area/areas/ruins/beachplanet.dm]: I feel like quirky cavern isn't the best name for this

File: _maps/RandomRuins/BeachRuins/beach_treasure_cove.dmm [hunk 3]
```diff
@@ -0,0 +1,3215 @@
...
+	pixel_y = 8;
+	pixel_x = 10
+	},
+/obj/item/radio{
+	desc = "An old handheld radio. You could use it, if you really wanted to.";
+	icon_state = "radio";
+	name = "old radio";
+	pixel_x = -5;
+	pixel_y = 2
+	},
```
Comment (Erikafox) [inline _maps/RandomRuins/BeachRuins/beach_treasure_cove.dmm:1122]: you use this a lot, you should make another pr subtyping it and updating your maps to use that subtype.

**Review by Erikafox — COMMENTED** [review]
map review later

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
I'd rename - redesc these. Perhaps some sort of "Water Boat"

StrongDMM_z3OLJRlzpM
Have the fuk off sign out front and make the turret function

File: code/game/objects/items/devices/radio/radio.dm [hunk 4]
```diff
@@ -432,3 +432,9 @@
 /obj/item/radio/off	// Station bounced radios, their only difference is spawning with the speakers off, this was made to help the lag.
 	listening = 0			// And it's nice to have a subtype too for future features.
 	dog_fashion = /datum/dog_fashion/back
+
+
+/obj/item/radio/old
+	name = "Old Radio"
```
Comment (Erikafox) [inline code/game/objects/items/devices/radio/radio.dm]: old radio, no capitals. Gotta keep things consistent.

**Comment by goober3** [comment]
`//seems quite simple to me`
You

**Review by ZephyrTFA — APPROVED** [review]

**Review by Erikafox — APPROVED** [review]

**Pull request merged**